consensus/types/src/chain_spec.rs

```diff
@@ -642,10 +639,8 @@ impl YamlConfig {
     }
 
     pub fn apply_to_chain_spec<T: EthSpec>(&self, chain_spec: &ChainSpec) -> Option<ChainSpec> {
```

Comment:
I notice we don't check `config_name` in this function.

I think this is fine, it might be nice to name them different things but have the same values. IMO we don't *really* care about the name.

Happy to leave this as-is.

Reply:
Yeah, part of the problem with that would be backwards compatibility, the Altona and Medalla configs don't have the `config_name` param at all (I guess we could default to `mainnet`). But happy to leave as-is.